css-color/Overview.bs

```diff
@@ -1717,7 +1717,7 @@ Profiled, Device-dependent Colors</h2>
 	or an RGB colorspace (such as ProPhoto <!-- ref -->, <!-- would be good to mention AdobeRGB 1998 here too, need to check trademark and copyright issues first --> widely used by photographers), or any other color or monochrome output device which has been characterized.
 	In addition, for convenience,
 	CSS provides two predefined RGB color spaces:
-	p3d65 [[!DCI-P3]],
+	display-p3 [[!DCI-P3]],
```

Comment:
RP 431-2 specifies a transfer function (gamma 2.6 and absolute XYZ coordinates) and white point there are no consistent with the use in this document. Also note that the term "p3" is not used once in RP 431-2.

At the very least, only Annex C.1 of RP 431-2 should be referenced. Ideally, RP 431-2 would not be normatively referenced, and instead the document would merely mention that the selected primaries happen to be identical to those specified in RP 431-2.